Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gcr.io/distroless/static
+
+COPY ./prometheus_bigquery_remote_storage_adapter /bigquery_remote_storage_adapter
```

Comment:
You might want to consider using a multi-stage build.

Here is an example Dockerfile can can start from.
```
# Step 1: build the binary
FROM golang:1.13.6 AS build_deps

WORKDIR /build
COPY go.mod go.sum ./
RUN go mod download

FROM build_deps AS builder

WORKDIR /build
COPY . .

RUN make clean
RUN make build

# Step 2: build small image
FROM gcr.io/distroless/static:nonroot

COPY --from=builder /build/prometheus_bigquery_remote_storage_adapter /

CMD ["/prometheus_bigquery_remote_storage_adapter"]
```

Reply:
Why are you suggesting to build via a dockerfile? Right now I have goreleaser build the image directly.

Reply:
My suggestion is to make sure that the local development workflow is frictionless. I haven't used goreleaser to build container images.

Reply:
We're good then. It works great even locally. I'll have to update the docs a bit, but then we should be good.